src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -791,31 +820,52 @@ def collect_periscope_logs(resource_group_name, name, storage_account_name=None,
 
             print()
             print(f"{colorama.Fore.GREEN}Deploying diagnostic container on the K8s cluster...")
-            subprocess_cmd = kubectl_prior + ["apply", "-f", temp_yaml_path, "-n", "aks-periscope"]
+            subprocess_cmd = kubectl_prior + ["apply", "-f", temp_yaml_path, "-n", "aks-periscope", "--validate=false"]
```

Comment:
what does --validate=flase do?

Reply:
I added the field --validate=false because in the event that the user doesn't specify a storage account or SAS token and we apply the periscope yaml, we will end up with a validation error. Adding --validate=false helps mitigate the validation error.